component/template.hbs

```diff
@@ -111,18 +111,50 @@
         expand=(action expandFn)
         expandOnInit=false
       }}
-        <div class="row">
-          <div class="col span-6">
+        <div class="row inline-form">
+          <div class="col span-2 col-inline">
             <label class="acc-label">
               Root Volume (in GB)
             </label>
-            <div>
-              {{input
-                type="text"
-                value=config.volumeSizeGb
-                classNames="form-control"
-              }}
-            </div>
+          </div>
+          <div class="col span-10">
+            {{input
+                    type="text"
+                    value=config.volumeSizeGb
+                    classNames="form-control"
+            }}
+          </div>
+        </div>
+        <hr/>
+        <div class="row inline-form">
+          <div class="col span-2 col-inline">
+            <label class="acc-label">
+              SSD Volumes (in GB)
+            </label>
+          </div>
+          <div class="col span-10">
+            {{form-value-array
+                    initialValues=config.volumeSsd
+                    addActionLabel="generic.add"
```

Comment:
It looks like the requirement for externalised/transalted strings is hard-coded into `form-value-array`. It's not a big deal, but we could make the UI a bit nicer by using our own strings instead of relying on the generic ones.

Do you know if it is possible to supply translations within a driver?

Reply:
I couldn't figure out a way to supply translations within the driver, that's why I choose this way. But I agree, not the nicest way!

Reply:
Okay. You came to the same conclusion then. Leaving this as it is for now.